app/pages/AssetMoneyValuePage.scala

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright 2019 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package pages
+
+import play.api.libs.json.JsPath
+
+case object AssetMoneyValuePage extends QuestionPage[String] {
+  def unapply(page: Page): Option[Any] = ???
+
+
+  override def path: JsPath = JsPath \ toString
```

Comment:
Do we need to scope the path in mongo to `JsPath \ Assets \ toString` where normally we would index this as well? Due to only having 1 monetary amount I think we should index this to conform with logic that @JoshuaH01 started to implement in the page before.